### PR TITLE
[16.0] [MIG] web_sheet_full_width

### PR DESCRIPTION
**Pull request by hugosantosred**
Migration of web_sheet_full_width to v16

#2309 

Works as expected without other changes.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Not sure if we should pack all the moddings to the default responsive theme in one module:

- This full width.
- App switcher.
- etc

@SplashS what do you think? Are you working on something?

**Comment by LoisRForgeFlow** [comment]
@pedrobaeza This module is still useful to use in conbination with EE.

**Review by LoisRForgeFlow — APPROVED** [review]
Functional test :+1:

**Review by tarteo — CHANGES_REQUESTED** [review]
Installed locally, but I don't see any changes to the width of the sheet.

image

**Comment by flotho** [comment]
any chance to retrigger the runboat build

**Comment by baimont** [comment]
> Installed locally, but I don't see any changes to the width of the sheet.
> 
> image

Hi there

Installed locally on an enterprise version.
It works for me.

For a large width I get the same result as @tarteo, which is the standard way. 
image

But when you reduce the width so that the chatter is displayed below the sheet, the sheet takes all the width of the window.
image

This is different from the standard.
image

So functional test ok for me! Thanks for this pr.

File: web_sheet_full_width/README.rst [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fweb-lightgray.png?logo=github
+    :target: https://github.com/OCA/web/tree/15.0/web_sheet_full_width
```
Comment (baimont) [inline web_sheet_full_width/README.rst:17]: I don't know when and how the 15.0 mentions will be replaced by 16.0, maybe when this is merged?

**Review by baimont — APPROVED** [review]

**Review by hieulucky111 — APPROVED** [review]
LGTM

**Comment by flotho** [comment]
runboat stucked, any chance to retrigger an event for testing on runboat

**Comment by hugosantosred** [comment]
@tarteo That behaviour with the chatter on the right is the default for large width. When web_chatter_position gets migrated this behaviour could be configured to show the chatter always on the bottom.

I've just tried by removing the chatter manually in a large width:

- With the standard behaviour
 
image

- With web_sheet_full_width installed

image

File: web_sheet_full_width/README.rst (hunk 1 above)
Comment (hugosantosred) [reply web_sheet_full_width/README.rst:17]: I don't know either. I'm going to check it in the case I missed to change it manually

Comment (hugosantosred) [reply web_sheet_full_width/README.rst:17]: The README.rst its generated again when is merged by oca-bot so that mentions to 15.0 will be replaced then.

**Review by ivantodorovich — APPROVED** [review]

**Comment by liebana** [comment]
Functionally tested, LGTM

**Review by antoniocanovas — APPROVED** [review]
Functional review Ok

**Pull request merged**